.github/workflows/release.yml

```
@@ -13,10 +13,12 @@ jobs:
         with:
           fetch-depth: 0
           persist-credentials: false
-      - name: Setup Node.js v14
+
+      - name: Setup Node.js
         uses: actions/setup-node@v2
```

actionlint (1.7.7)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)